src/easydmp/dmpt/models.py

```diff
@@ -1510,6 +1515,12 @@ def frame_canned_answer(self, answer, frame=True):
             result = self.framing_text.format(answer)
         return mark_safe(result)
 
+    def is_optional_section_question(self):
```

Comment:
@hmpf Shouldn't we use the get_optional_section_question() you add in #128 here?